code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -20,23 +20,29 @@
 
 	var/process_type //Determines what to do when process_scan() recieves a target. See process_scan() for details.
```

Comment:
Autodoc this please!

Reply:
Im gonna autodoc you 
[bd8a88f](https://github.com/ParadiseSS13/Paradise/pull/23162/commits/bd8a88f0743a43736235b45a9e727dc868e5a462)